docs/bin/build.js

```diff
@@ -16,7 +16,7 @@ const run = require('../lib/build.js')
 const { paths } = require('../lib/index')
 
 run(paths)
-  .then((res) => console.log(`Wrote ${res.length} files`))
+  .then((res) => console.error(`Wrote ${res.length} files`))
```

Comment:
This change was made so that our `prepack` script doesn't log anything to stdout, so that a smoke-test could be added that running `node . pack --json` only ends up with json parseable output on stdout.

This highlights an important note about this change, as it only redirects the banner from `@npmcli/run-script` to stderr when `--json` is set. The sub-process is spawned with `stdio: inherit` so if that process sends output to stdout it could still break in json mode. This is not something `npm` should change since we want to allow spawned process to write to whatever stream they want.

If a user really wants to ensure that the script cannot output anything, they should also use `--foreground-scripts=false`.